applications/SolversApplication/linear_system/linear_solvers/feast_solver.hpp

```diff
@@ -0,0 +1,531 @@
+//
+//   Project Name:        KratosSolversApplication $
```

Comment:
@josep-m-carbonell For example this is literally the duplication of existing code, if an error is found in here or in the original one the code should be corrected in two different places, this is against the principals of code development